src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -251,6 +251,9 @@ public void StartAdbServer()
         {
             throw new AdbFailureException("Failed to start the ADB server");
         }
+
+        // Android logs can unnecessarily hide log entires, so disable
+        DisableChatty();

```

Comment:
I don't think we should do this in `StartAdbServer()` since we don't know whether the device is connected at that point.

I'd move it directly into `ClearAdbLog()` instead.